assets/img/GRCLogoOnly.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 433.01 500"><defs><linearGradient id="a" x1="250" y1="4.41" x2="250" y2="501" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#7f00f7"/><stop offset="0.54" stop-color="#41007f"/><stop offset="1" stop-color="#41007f"/></linearGradient><linearGradient id="b" x1="250" y1="4.41" x2="250" y2="501" xlink:href="#a"/></defs><polygon points="2.44 126.41 216.5 2.82 430.57 126.41 430.57 373.59 216.5 497.18 2.44 373.59 2.44 126.41" fill="#fff"/><path d="M342.46,82.09a342.06,342.06,0,0,1-40.14,28.45c-13.31,8.18-27,15.81-40.27,23.23q-6.15,3.43-12.05,6.78C189.32,175,150.28,202.91,150.28,250c0,39.79,28,65.71,73.4,93.44,5.18,3.17,10.62,6.36,16.24,9.6-2,1.12-4,2.22-6,3.35-13.61,7.54-27.67,15.33-41.6,23.89q-2.09,1.28-4.15,2.57c-4.91-3.2-9.77-6.46-14.49-9.87-40-28.86-71.52-65.93-71.52-123,0-68.08,44.93-108,95.53-139.46,14.13-8.79,28.69-16.91,42.85-24.8L250,80.42c16.25-9.18,30.83-17.85,43.65-26.51L250,28.71,206.34,53.92c5.49,3.71,11.23,7.42,17.39,11.17,5.18,3.16,10.6,6.35,16.21,9.59L235.75,77c-14.21,7.92-28.9,16.12-43.24,25-1.39.86-2.74,1.72-4.09,2.58-4.91-3.18-9.77-6.43-14.5-9.81-5.67-4.06-11.12-8.33-16.38-12.74L58.36,139.36V360.64l98.58,56.92a332.75,332.75,0,0,1,40.5-28.78c13.6-8.36,27.6-16.12,41.22-23.66q5.79-3.21,11.34-6.33c29-16.35,53.12-31.21,70.34-47.41H217.12l30.95-53.71H397.65a128.92,128.92,0,0,1-14.39,53.71c-17.24,33.09-47.76,57.06-80.69,77.4-14,8.66-28.46,16.66-42.51,24.44-3.42,1.89-6.76,3.76-10.06,5.61-16.54,9.28-31.34,18.07-44.31,26.87L250,471.29l44.33-25.6c-5.65-3.83-11.58-7.67-17.95-11.54q-7.8-4.74-16.28-9.58l4.71-2.63c14.09-7.81,28.67-15.88,42.91-24.67,1.36-.84,2.74-1.72,4.13-2.59,4.89,3.17,9.73,6.39,14.43,9.76,5.82,4.17,11.4,8.56,16.78,13.12l98.58-56.92V139.36Zm4.87,148.73c-7.65-29.64-33.4-51.64-71.18-74.9-5.15-3.17-10.55-6.37-16.13-9.62,2.26-1.27,4.49-2.53,6.81-3.82,12.63-7,26.95-15,40.62-23.45q2.06-1.26,4.11-2.55c4.9,3.21,9.74,6.49,14.45,9.9,35.44,25.74,64.29,57.88,70.54,104.44Z" transform="translate(-33.49)" fill="url(#a)"/><path d="M250,500,33.49,375V125L250,0,466.51,125V375ZM44,369,250,487.92,456.05,369V131l-206-119L44,131Z" transform="translate(-33.49)" fill="url(#b)"/></svg>
```

Comment:
Can compress this down a little bit more. Also would be good to make all these new files use lowercase names

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 433.01 500"><defs><linearGradient id="A" x1="250" y1="4.41" x2="250" y2="501" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#7f00f7"/><stop offset=".54" stop-color="#41007f"/></linearGradient><linearGradient id="B" xlink:href="#A" /></defs><path fill="#fff" d="M2.44 126.41L216.5 2.82l214.07 123.59v247.18L216.5 497.18 2.44 373.59V126.41z"/><g transform="translate(-33.49)"><path d="M342.46 82.09a342.06 342.06 0 0 1-40.14 28.45c-13.31 8.18-27 15.81-40.27 23.23q-6.15 3.43-12.05 6.78C189.32 175 150.28 202.91 150.28 250c0 39.79 28 65.71 73.4 93.44 5.18 3.17 10.62 6.36 16.24 9.6l-6 3.35c-13.61 7.54-27.67 15.33-41.6 23.89q-2.09 1.28-4.15 2.57c-4.91-3.2-9.77-6.46-14.49-9.87-40-28.86-71.52-65.93-71.52-123 0-68.08 44.93-108 95.53-139.46 14.13-8.79 28.69-16.91 42.85-24.8l9.46-5.3c16.25-9.18 30.83-17.85 43.65-26.51L250 28.71l-43.66 25.21c5.49 3.71 11.23 7.42 17.39 11.17l16.21 9.59-4.19 2.32c-14.21 7.92-28.9 16.12-43.24 25l-4.09 2.58c-4.91-3.18-9.77-6.43-14.5-9.81-5.67-4.06-11.12-8.33-16.38-12.74l-99.18 57.33v221.28l98.58 56.92a332.75 332.75 0 0 1 40.5-28.78c13.6-8.36 27.6-16.12 41.22-23.66q5.79-3.21 11.34-6.33c29-16.35 53.12-31.21 70.34-47.41H217.12l30.95-53.71h149.58a128.92 128.92 0 0 1-14.39 53.71c-17.24 33.09-47.76 57.06-80.69 77.4-14 8.66-28.46 16.66-42.51 24.44L250 418.83c-16.54 9.28-31.34 18.07-44.31 26.87L250 471.29l44.33-25.6a426.58 426.58 0 0 0-17.95-11.54q-7.8-4.74-16.28-9.58l4.71-2.63c14.09-7.81 28.67-15.88 42.91-24.67l4.13-2.59c4.89 3.17 9.73 6.39 14.43 9.76 5.82 4.17 11.4 8.56 16.78 13.12l98.58-56.92V139.36zm4.87 148.73c-7.65-29.64-33.4-51.64-71.18-74.9l-16.13-9.62 6.81-3.82c12.63-7 26.95-15 40.62-23.45q2.06-1.26 4.11-2.55c4.9 3.21 9.74 6.49 14.45 9.9 35.44 25.74 64.29 57.88 70.54 104.44z" fill="url(#A)"/><path d="M250,500,33.49,375V125L250,0,466.51,125V375ZM44,369,250,487.92,456.05,369V131l-206-119L44,131Z" fill="url(#B)"/></g></svg>
```